app/helpers/govspeak_helper.rb

```diff
@@ -38,9 +38,10 @@ def bare_govspeak_edition_to_html(edition)
   def govspeak_html_attachment_to_html(html_attachment)
     # HTML attachments can embed images - but not attachments - from their parent Edition
     images = prepare_images(html_attachment.attachable.try(:images) || [])
+    locale = html_attachment.translated_locales.first
```

Comment:
The only "?" here (from a cursory look) is the `.last` - do we ever expect to have to handle more than one locale in this helper? Are we missing something, eg potentially choosing the 'wrong' locale?